cpp-client/README.md

```diff
@@ -44,11 +44,13 @@ C++ compiler and tool suite (cmake etc).
 
    Example:
    ```
-   mkdir -p $HOME/dhcpp
```

Comment:
I can't put this comment on the line I want, but about 15 lines above, please delete the backticks around the two HTTP URLs so they render as Markdown URLs and not emphasized text.

Put another way, change
~~~
`https://github.com/deephaven/deephaven-base-images/blob/main/cpp-client`
~~~
and
~~~
`https://github.com/deephaven/deephaven-base-images/raw/main/cpp-client/build-dependencies.sh`
~~~

to

~~~
https://github.com/deephaven/deephaven-base-images/blob/main/cpp-client
~~~
and
~~~
https://github.com/deephaven/deephaven-base-images/raw/main/cpp-client/build-dependencies.sh
~~~